twake/backend/node/src/services/tags/entities/tags.ts

```diff
@@ -0,0 +1,34 @@
+import { Type } from "class-transformer";
+import { Column, Entity } from "../../../core/platform/services/database/services/orm/decorators";
+
+export const TYPE = "tags";
+/**
+ * Table user-notification-badges
+ */
+@Entity(TYPE, {
+  primaryKey: [["company_id"], "name", "colour"],
+  type: TYPE,
+})
+export class Tags {
+  /**
```

Comment:
- The company_id column seems to be missing
- the primary-key can't be this, think "unique sorted identifier of your entity", here `[company_id], tag_id` is what you want